dbcon/joblist/tupleunion.h

```diff
@@ -122,8 +122,8 @@ class TupleUnion : public JobStep, public TupleDeliveryStep
   };
 
   void getOutput(rowgroup::RowGroup* rg, rowgroup::Row* row, rowgroup::RGData* data);
-  void addToOutput(rowgroup::Row* r, rowgroup::RowGroup* rg, bool keepit, rowgroup::RGData& data);
-  void normalize(const rowgroup::Row& in, rowgroup::Row* out);
+  void addToOutput(rowgroup::Row* r, rowgroup::RowGroup* rg, bool keepit, rowgroup::RGData& data, uint32_t& tmpOutputRowCount);
+  void normalize(const rowgroup::Row& in, rowgroup::Row* out, std::vector<std::function<void(const rowgroup::Row& in, rowgroup::Row* out, uint32_t col)>>& normalizeFunctions);
```

Comment:
Can you create a typedef for this vector in the global namespace in this .h file and reuse it here as well as in the .cpp?
```c++
using normalizeFunctionsT = std::vector<std::function<void(const rowgroup::Row& in, rowgroup::Row* out, uint32_t col)>>;
```

Also, I think `normalizeFunctions` should be a const ref here:
```c++
void normalize(const rowgroup::Row& in, rowgroup::Row* out, const normalizeFunctionsT& normalizeFunctions);
```